test/unit/nedap/utils/test/matchers.cljc

```diff
@@ -0,0 +1,20 @@
+(ns unit.nedap.utils.test.matchers
```

Comment:
strictly this is an integration test, as we're testing integration with matcher-combinators.